rocketpy/environment/environment.py

```diff
@@ -1507,7 +1507,7 @@ def wind_heading_func(h):
         )
 
         def wind_direction(h):
-            return (wind_heading(h) - 180) % 360
+            return (wind_heading_func(h) - 180) % 360
```

Comment:
Was this a bug? Or was this changed just to speed up code?

Both versions should return the same result

Reply:
sorry this is one small flaw of this PR: some changes I'm doing are more to improve readability, while only others are actually speeding up the code. 

I'm having a hard time trying to separate the modifications as much as possible, I appreciate reviewers patience for cases like this one.